plugins/module_utils/copy.py

```diff
@@ -64,6 +64,50 @@ def __init__(self, module, client):
         self.container_arg = {}
         if module.params.get("container"):
             self.container_arg["container"] = module.params.get("container")
+        self.check_mode = self.module.check_mode
+
```

Comment:
Since the class doesn't use `self`, this can be either a `@classmethod` or a `@staticmethod`. This will simplify the unit-test writing a bit.